lib/internal/streams/readable.js

```diff
@@ -404,7 +437,7 @@ function computeNewHighWaterMark(n) {
 function howMuchToRead(n, state) {
   if (n <= 0 || (state.length === 0 && state.ended))
```

Comment:
Can you please replace `ended` with bitwise operation?

And there maybe some more left :)